src/aggregation/mod.rs

```diff
@@ -132,6 +132,7 @@
 //!             bucket_agg: BucketAggregationType::Range(RangeAggregation{
 //!                 field: "score".to_string(),
 //!                 ranges: vec![(3f64..7f64).into(), (7f64..20f64).into()],
+//!                 keyed: false,
```

Comment:
```suggestion
```

Reply:
@PSeitz 
Could we do that?
Just removing the field will cause compiler error.
```
missing field `keyed` in initializer of `RangeAggregation`
```

Reply:
ah okay there's no `..Default::default`, we can keep it here